### DISCUSSION TITLE
Thanks for making this

### DISCUSSION THREAD

**Discussion by gmadharh** [gxy5202/parcel-namer-hashless#11] [General]
Currently working on a chrome extension and this was exactly what I needed, was very annoying to fix all of these file names and paths. Just wanted to show my appreciation for making this package! 😊

**Comment by gxy5202** [comment]
> Currently working on a chrome extension and this was exactly what I needed, was very annoying to fix all of these file names and paths. Just wanted to show my appreciation for making this package! 😊

Thanks for your compliment.Yes, i just think probably there are some developers also meeting hash problems like me. So i published this package to the NPM. 😉
